apps/condo/domains/resident/tasks/sendResidentMessage.task.js

```diff
@@ -0,0 +1,9 @@
+const { createTask } = require('@open-condo/keystone/tasks')
+
+const { sendResidentMessage } = require('./helpers')
```

Comment:
We don't use such a file structure. remove `helpers` folder and write logic inside tasks.

Reply:
In current PR I'd move helper inside task, but there are other helpers for regular cron tasks that we've discussed and agreed to move to Billing domain in separate PR.

Reply:
done